src/setUpTests.js

```diff
@@ -1,15 +0,0 @@
-import configureStore from "redux-mock-store";
```

Comment:
delete one file, another one appears... delete that file, the other one appears 👻 👻 👻